UI/src/sass/contact-banner.scss

```diff
@@ -0,0 +1,4 @@
+﻿.app-contact-banner {
+    background-color: govuk-colour("blue");
+    color: govuk-colour("white") !important;
```

Comment:
This needs to specify height 70px as per the design details. There will also need to be padding or vertical centring of the text to match the visual design.